__tests__/users.ts

```diff
@@ -1159,12 +1169,62 @@ describe('user company', () => {
         variables: { email: 'u1@com3.com' },
       });
       expect(res.errors).toBeFalsy();
-      const row = await con.getRepository(UserCompany).findOneBy({
+      const row = await con.getRepository(UserCompany).findOneByOrFail({
         email: 'u1@com3.com',
       });
       expect(row.verified).toBeFalsy();
       expect(row.code.length).toEqual(6);
     });
+
+    it('should send verification email to user if email is not verified', async () => {
+      loggedUser = '1';
+      const res = await client.query(QUERY, {
+        variables: { email: 'u1@com4.com' },
+      });
+      expect(res.errors).toBeFalsy();
+      const row = await con.getRepository(UserCompany).findOneByOrFail({
+        email: 'u1@com4.com',
+      });
+      expect(row.verified).toBeFalsy();
+      expect(row.code.length).toEqual(6);
+      expect(row.companyId).toEqual(null);
+
+      expect(sendEmail).toHaveBeenCalledTimes(1);
+      expect(sendEmail).toHaveBeenCalledWith({
+        to: 'u1@com4.com',
+        message_data: {
+          code: expect.any(String),
+        },
+        identifiers: {
+          id: loggedUser,
+        },
+        transactional_message_id: '51',
```

Comment:
Use enum value here in case its changed so test dont fail?

Reply:
Added